docs/technical_reference/unit_models/ion_exchange_0D.rst

```diff
@@ -0,0 +1,318 @@
+   "Column volume calculated from bed volume", ":math:`V_{col} = H \frac{V_{bed}}{Z}`"
+   "Column volume calculated from column diameter", ":math:`V_{col} = \pi (\frac{D_{col}}{2})^{2} H`"
+   "Column diameter calculation", ":math:`(\frac{D_{col}}{2})^{2} = (\frac{H}{2X})^{2}`"
+   "Fluid mass transfer coeff", ":math:`k_{f} = \frac{D Sh}{d}`"
+   "Rate coefficient", ":math:`k = 6 \frac{(1-\epsilon)k_{f}}{\rho_{b}d}`"
+   "Height of transfer unit", ":math:`HTU = \frac{u_{bed}}{\rho_{b}k}`"
+   "Partition ratio", ":math:`\Lambda = \frac{q_{eq} \rho_{b}}{ñ_{in}}`"
+   "Left hand side of constant pattern solution", ":math:`lh = N(\tau - 1)`"
+   "Right hand side of constant pattern solution", ":math:`lh = 1 + \frac{\log{(C_{b}/C_{0})} - La \log{(1 - C_{b}/C_{0})}}{1 - La}`"
+   "Dimensionless time", ":math:`\tau = (\frac{u_{inter}t_{breakthru} \epsilon}{Z} - \epsilon) / \Lambda`"
+   "Number of mass-transfer units", ":math:`N = \frac{k_{f}a_{s}Z}{u_{bed}}`"
+   "Flow conservation", ":math:`Q_{p, in} - \frac{Q_{bw}t_{bw} + Q_{rinse}t_{rinse}}{t_{cycle}} = Q_{p, out} - \frac{Q_{regen}t_{regen}}{t_{cycle}}`"
+   "Influent total mass of ion", ":math:`M_{in} = Q_{p, in}t_{breakthru}ñ_{in}`"
+   "Removed total mass of ion", ":math:`M_{rem} = V_{bed}q_{eq}n_{op} \rho_{b}`"
+   "Mass of ion in effluent", ":math:`M_{out} = M_{in} - M_{rem}`"
+   "Steady-state effluent concentration (for target ion)", ":math:`ñ_{out} = \frac{M_{out}}{Q_{p, in}t_{breakthru}}`"
+   "Steady-state effluent concentration", ":math:`ñ_{out} = ñ_{in}`"
+   "Steady-state regen concentration (for target ion)", ":math:`ñ_{regen} = \frac{M_{rem}N_{regen}}{Q_{p, regen}t_{regen}}`"
+   "Steady-state regen concentration", ":math:`ñ_{regen} = 0`"
+   "Cycle time", ":math:`t_{cycle} = t_{breakthru} + t_{waste}`"
+   "Waste time", ":math:`t_{waste} = t_{regen} + t_{bw} + t_{rinse}`"
+   "Regen volumetric flow rate", ":math:`Q_{p, regen} = \frac{Q_{p, in}N_{regen}}{R}`"
+   "Regen pump power", ":math:`P_{regen} = \frac{9.81 \rho_{in} 0.70325P_{drop}Q_{p, regen}}{\eta}`"
+   "Bed expansion fraction from backwashing (T = 20C)", ":math:`X_{expan} = H_{expan,A} + H_{expan,B}u_{bw} + H_{expan,C}u_{bw}^{2}`"
+   "Bed expansion from backwashing", ":math:`H_{expan} = X_{expan}Z`"
+   "Backwashing flow rate", ":math:`Q_{bw} = u_{bw} \frac{V_{bed}}{Z}n_{op}`"
+   "Backwash pump power", ":math:`P_{bw} = \frac{9.81 \rho_{in} 0.70325P_{drop}Q_{bw}}{\eta}`"
+   "Rinse time", ":math:`t_{rinse} t_{contact} + N_{rinse}`"
+   "Rinse flow rate", ":math:`Q_{rinse} = u_{bed} \frac{V_{bed}}{Z}n_{op}`"
+   "Rinse pump power", ":math:`P_{rinse} = \frac{9.81 \rho_{in} 0.70325P_{drop}Q_{rinse}}{\eta}`"
+   "Main pump power", ":math:`P_{main} = \frac{9.81 \rho_{in} 0.70325P_{drop}Q_{p, in}}{\eta}`"
+   "Pressure drop (T = 20C)", ":math:`P_{drop} = Z(P_{drop,A} + P_{drop,B}u_{bed} + P_{drop,C}u_{bed}^{2})`"
+   "Total column volume required", ":math:`V_{col, tot} = n_{op}V_{col}`"
+
+
+References
```

Comment:
Other references to add: 

        # LeVan, M. D., Carta, G., & Yon, C. M. (2019).
        # Section 16: Adsorption and Ion Exchange.
        # Perry's Chemical Engineers' Handbook, 9th Edition.

        # Inamuddin, & Luqman, M. (2012).
        # Ion Exchange Technology I: Theory and Materials.